src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -392,6 +411,30 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 		return !!isFullyRendered;
 	}
 
+	private renderTreeData(data: IChatResponseProgressFileTreeData, element: ChatTreeItem, disposables: DisposableStore, templateData: IChatListItemTemplate): { element: HTMLElement; dispose: () => void } {
+		const ref = this._treePool.get();
+		const tree = ref.object;
+
+		const didOpenListener = tree.onDidOpen((e) => {
+			if (e.element && !('children' in e.element)) {
+				this.commandService.executeCommand('vscode.open', e.element.uri);
+			}
+		});
+
+		tree.setInput(data).then(() => {
+			tree.layout();
+			this._onDidChangeItemHeight.fire({ element, height: templateData.rowContainer.offsetHeight });
```

Comment:
It seems like this event should be fired when you collapse/expand a tree node, am I missing where that happens?